com.st.STM32CubeMX.yaml

```diff
@@ -0,0 +1,44 @@
+app-id: com.st.STM32CubeMX
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: stm32cubemx
+finish-args:
+  - --socket=x11
+  - --share=network
+  - --share=ipc
+  - --filesystem=home
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/installjdk.sh
+
+  - name: stm32cubemx
+    buildsystem: simple
+    build-options:
+      no-debuginfo: true
+      no-debuginfo-compression: true
```

Comment:
I have to use them because the file /app/stm32cubemx/STM32CubeMX is both an elf and a jar file. If you remove these options, the jar will stop running.